.github/workflows/nodejs.yml

```diff
@@ -0,0 +1,30 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Node.js CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [10.x, 12.x]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: clone the repo
+      run: git clone https://github.com/sumangala028/learn-github-action.git
+    - name: execute the file
+      run: npm install -g create-react-app
```

Comment:
What is the use of this line?

Reply:
These steps are to create a react app

Reply:
why do you want to create react app here? Why did you clone before?

Reply:
Oops. So after the clone step, we can just have a command to run the react project.
Right?

Reply:
yes that's enough !!

Reply:
So update the yml file and push again

Reply:
Okay.